### DISCUSSION TITLE
How to patch boot.img ?

### DISCUSSION THREAD

**Discussion by USAESAM** [bmax121/APatch#703] [Q&A] [answered]
How to patch boot.img ?

**Comment by MagicianGuo** [comment]
> How to patch boot.img ?

Extract boot.img from the ROM package provided by the mobile phone manufacturer, and then follow the APatch official tutorial operation.

从手机厂商官方提供的ROM包里面提取出boot.img，然后按照APatch官方教程操作。

**Comment by USAESAM** [comment]
Any tutorial video

**Reply by marat2509** [reply, marked as answer]
https://apatch.dev/install.html
